### ISSUE TITLE
[doc] nn.Module.forward documentation unclear

### ISSUE THREAD

**Issue by henon** [pytorch/pytorch#22220]
## 📚 Documentation

This note is totally unclear to me. What does it mean? I don't know what should be called afterward and what is meant by former and latter.

![image](https://user-images.githubusercontent.com/44090/60126626-b6d3e100-978e-11e9-9d76-d7df1f7415dc.png)

**Comment by SsnL** [comment]
module_instance(inputs)

**Comment by henon** [comment]
I searched the PyTorch docs and didn't find any hits for `module_instance`. Also class Module doesn't seem to contain such a method when looking at the source. What am I missing?

**Comment by SsnL** [comment]
I was just referring to a generic instance of the module class..

**Comment by yf225** [comment]
@henon An example for this would be:
```python
m = torch.nn.Linear(10, 20)
input = torch.randn(10)
output = m(input)
```
We should add a simple example in the `nn.Module.forward` doc about this.

**Comment by henon** [comment]
@yf225 thanks for the explanation. So is it true to say that any Module is in fact a Callable?

**Comment by apaszke** [comment]
@henon yes, every module is a callable, and calling it is (modulo some bookkeeping that we do automatically) equivalent to doing `module.forward(input)` (**which you should never do!!**).
